src/data/set/intervals/ord_connected.lean

```diff
@@ -0,0 +1,108 @@
+/-
+Copyright (c) 2020 Yury G. Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Yury G. Kudryashov
+-/
+import data.set.intervals.unordered_interval
+import data.set.lattice
+
+/-!
+# Order-connected sets
+
+We say that a set `s : set α` is `ord_connected` if for all `x y ∈ s` it includes the
+interval `[x, y]`. If `α` is a `conditionally_complete_linear_order` with the `order_topology`,
```

Comment:
don't you also need the order to be dense?

Reply:
Yes, sure.